log.go

```diff
@@ -1,65 +1,139 @@
 package fnlog
 
 import (
+	"context"
 	"fmt"
 	"io"
 	"os"
 )
 
-func Info(args ...interface{}) {
+type standard struct {
+	Level   LogLevel
+	Context context.Context
+}
+
+// LogWithoutContext - get log instance
+func LogWithoutContext(level LogLevel) Logger {
```

Comment:
I don't think LogWithoutContext should be work as global log, I've already implement it in log function as .Info, .Debug if you not send ctx or key in args[0].

Reply:
an example use is 
fnlog. LogWithoutContext(fnlog.InfoLevel).Info(ctx, "debug message")
it can use dependency injection

I think it should not use as global because it can't set log level, besides you have a global config.